template-manager-cmtemplate/src/main/resources/definitions/cm-cloud-storage-location-specification.json

```diff
@@ -30,16 +30,6 @@
       "requiredForAttachedCluster": false,
       "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
     },
-    {
```

Comment:
I thought we need warehouse dir

Reply:
In CDH, As part of [DWX-245](https://jira.cloudera.com/browse/DWX-245) spark is being configured to always create HMS external tables and hence uses HMS warehouse directory. So spark warehouse directory should not be initialized.